shared/minime/contracts/MiniMeToken.sol

```diff
@@ -192,7 +192,7 @@ contract MiniMeToken is Controlled {
         require((_to != 0) && (_to != address(this)));
         // If the amount being transfered is more than the balance of the
         //  account the transfer returns false
-        var previousBalanceFrom = balanceOfAt(_from, block.number);
+        uint256 previousBalanceFrom = balanceOfAt(_from, block.number);
```

Comment:
Could you revert the changes to the `MiniMeToken` here?

I'm not opposed to fixing the compiler warnings here, but we'd also like it to stay as closely as possible to the original version and we should have a separate discussion if we should just [update to `MMT_0.2`](https://github.com/giveth/minime/blob/master/contracts/MiniMeToken.sol).

Reply:
Sure thing!